### ISSUE TITLE
Kali Terminal error '1) no title' on Nexus 5 running 6.0.1

### ISSUE THREAD

**Issue by audibleblink** [offensive-security/kali-nethunter#353]
__Device__: Nexus 5
__ROM__:  Stock Hammerhead 6.0.1 (MMB29S)
__Root__: SuperSU-v2.67-20160121175247
__Kernel__: 3.4.0-gb7558f2

Noticed error when trying to run Nethunter Terminal and selecting `KALI` then again when trying to install/update packages to the chroot. 

Running `ls -la /system/bin/boot`

```sh
lrwxrwxrwx root     root              2015-01-01 07:09 bootkali -> /data/data/com.offsec.nethunter/files/scripts/bootkali
lrwxrwxrwx root     root              2015-01-01 07:09 bootkali_bash -> /data/data/com.offsec.nethunter/files/scripts/bootkali_bash
lrwxrwxrwx root     root              2015-01-01 07:09 bootkali_login -> /data/data/com.offsec.nethunter/files/scripts/bootkali_login
```

Checking symlink target existence with `ls -la /data/data/com.offsec.nethunter/files/scripts/boo*`

```sh
-rwx------ u0_a83   u0_a83      14058 2016-01-23 13:00 bootkali
-rwx------ u0_a83   u0_a83       2615 2016-01-23 13:00 bootkali_bash
-rwx------ u0_a83   u0_a83       2628 2016-01-23 13:00 bootkali_lo
```

FWIW: `/data/user/` is symlinked to `/data/data/`

**Comment by audibleblink** [comment]
From ADB shell

```
126|shell@hammerhead:/ $ su
root@hammerhead:/ # /system/bin/bootkali

chroot: can't execute '/bin/bash': Exec format error
```

**Comment by audibleblink** [comment]
Anything that uses `sudo $busybox chroot /data/local/nhsystem/kali-armhf /bin/bash` fails

**Comment by jmingov** [comment]
Hello, you flashed the nethunter zip from https://www.offensive-security.com/kali-linux-nethunter-download/?

**Comment by audibleblink** [comment]
Indeed I did, @jmingov

**Comment by jmingov** [comment]
Can you try to reinstall the chroot from nethunter app?

Just go to chroot manager in nethunter app and hit `remove chroot`, (that will remove the chroot after a reboot), the reinstall the chroot from app also  (use wifi ^^).

Keep me updated please, we never saw this error in nethunter.

Jaime

**Comment by audibleblink** [comment]
That did it. Works now. Fixes #354 too

**Comment by jmingov** [comment]
Nice ^^
